inspectors/epa.py

```diff
@@ -108,6 +108,9 @@ def report_from(tds, published_on_dt, year):
   if report_url:
     report_url = urljoin(BASE_URL, report_url)
 
+  if report_url == "http://www.nsf.gov/oig/USMFS05.pdf":
+    report_url = "http://www.nsf.gov/oig/_pdf/USMFS05.pdf"
+
```

Comment:
Looks like the NSF OIG asked nsf.gov to let HTTPS stay on for the OIG site, so these could be moved to `https://`.